starter_code/elevator.js

```diff
@@ -3,17 +3,68 @@ class Elevator {
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "";
+    this.timerId;
+    this.waitingList = []; //made request but not on yet
+    this.passengers = []; //currently in elevator
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+  start() {
+    this.timerId = setInterval(update, 1000);
```

Comment:
you could implement this with an arrow function: 
    **this.timer = setInterval(() => { this.update() }, 1000); :**